master/internal/core_experiment.go

```diff
@@ -293,7 +293,13 @@ func (m *Master) parseCreateExperiment(ctx context.Context, req *apiv1.CreateExp
 	}
 	workspaceID := resolveWorkspaceID(workspaceModel)
 	isSingleNode := resources.IsSingleNode() != nil && *resources.IsSingleNode()
-	poolName, _, err := m.ResolveResources(resources.ResourcePool(), resources.SlotsPerTrial(), workspaceID, isSingleNode)
+	isUnmanaged := req.GetUnmanaged()
+	slotsPerTrial := resources.SlotsPerTrial()
+	if isUnmanaged {
+		slotsPerTrial = 0
+	}
```

Comment:
if you're going to use this, you really dont need the temporary variable
```suggestion
	slotsPerTrial := resources.SlotsPerTrial()
	if req.GetUnmanaged() {
		slotsPerTrial = 0
	}
```